remix/app/helpers/button.helper.ts

```diff
@@ -0,0 +1,33 @@
+export function setButtonColor(color?: string) {
```

Comment:
I think this is more like a method to get something(a string that contains tailwind classes), more than a method to set something 😸 

Maybe `getButtonClassNames`? 🙈

Reply:
Yep, but I think `getButtonClassNames` is too general for me like I just want to get the color from the method, should we name it `getButtonColorClassNames` or `getButtonStyle`. 

What do you think is the best here, I'm Ok if you go with `getButtonClassNames` I can change it to. 😂

Reply:
I see. The `getButtonColorClassNames` is a bit long but it sounds best to me 😄

Reply:
ya I think so, how about `getButtonStyle` kinda like ok right? properly not the good

Reply:
It also sounds ok! As long as it start with `get`, I think its better 😄

Reply:
1. The switch is not really on the color but on the type. Yes it will change the color styles but from a developer point of view, it makes more sense to have `setButtonType('primary')`.
2. It might be best to colocate this code in the directory `AppLink` to keep all related code together 💡 I doubt that this code will be used somewhere else than in `AppLink`

Reply:
Well noted 👍🏼

Reply:
Improved here [b357dd3](https://github.com/nimblehq/js-framework-benchmark/pull/130/commits/b357dd39fc2364fe1acf62523fa9f7463e88caed)

Reply:
is it ok, if I still want to separate it from the `AppLink` directory, like this method might be used for another similar button component in the future?

Reply:
> this method might be used for another similar button component in the future

Well, it is not now 😅 So it can be moved once that case happens. Prefer not to over-optimize for unknown future needs. The codebase should reflect the current and future confirmed needs.

Reply:
Solved here: [67a8330](https://github.com/nimblehq/js-framework-benchmark/pull/130/commits/67a83308dcdfa4aa48b422d8e184137f5db93c77)